File: Python/List-2/sum67.py
# my solution
def sum67(nums):
	n = len(nums)
	total = i = 0
	while i < n:
		if nums[i] == 6:
			while nums[i] != 7:
				i += 1
		else:
			total += nums[i]
		i += 1
	return total

File: Java/Array-2/sum67.java
public int sum67(int[] nums) {
  int len = nums.length;
  int total =0, i = 0;
  while (i < len)
  {
    if (nums[i] == 6)
    {
      while (nums[i] != 7)
        i++;
      // at this point, we are at the 7
      // increasment by 1 to the next number
      i++;
    } else
    {
      total += nums[i];
      i++;
    }
    
  }
  return total;
}
